browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -3560,27 +3540,28 @@ index 3805f84245f6cb73c45844364d54f822b6beb26d..a6c62b3bcceaa054095cb9edb853de60
  {
      Document* document = &node->document();
      if (auto* templateHost = document->templateDocumentHost())
-@@ -2846,12 +2980,16 @@ RefPtr<Protocol::Runtime::RemoteObject> InspectorDOMAgent::resolveNode(Node* nod
+@@ -2848,12 +2982,17 @@ RefPtr<Protocol::Runtime::RemoteObject> InspectorDOMAgent::resolveNode(Node* nod
      if (!frame)
          return nullptr;
  
--    auto& state = *mainWorldExecState(frame);
--    auto injectedScript = m_injectedScriptManager.injectedScriptFor(&state);
+-    auto& globalObject = mainWorldGlobalObject(*frame);
+-    auto injectedScript = m_injectedScriptManager.injectedScriptFor(&globalObject);
 +    InjectedScript injectedScript;
 +    if (contextId) {
 +        injectedScript = m_injectedScriptManager.injectedScriptForId(*contextId);
 +    } else {
-+        injectedScript = m_injectedScriptManager.injectedScriptFor(mainWorldExecState(frame));
++        auto& globalObject = mainWorldGlobalObject(*frame);
```

Comment:
nit: would it make sense to keep `auto& globalObject = mainWorldGlobalObject(*frame);` outside the `if` and avoid modifications at line 3560 (just to keep our patch smaller) ?

Reply:
Sure, I will rework this part in the next roll.